test/ERC1155Mintable.test.ts

```diff
@@ -729,7 +780,7 @@ describe('ERC1155Mintable SDK', () => {
     };
 
     expect(approval).not.toThrow();
-    expect(contractFactoryMock).toHaveBeenCalledTimes(1);
+    expect(await contractFactoryMock).toHaveBeenCalledTimes(1);
```

Comment:
I think await can be removed here

Reply:
it fails if I do not wait for the promise